src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     <NoWarn>1718</NoWarn>
     <IncludePerformanceTests>true</IncludePerformanceTests>
     <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TestTFMs)'=='netcoreapp1.1'">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Shouldn't NuGetTargetMoniker already be set by the targetgroup?  If so, then just add a condition to the line above to either only set it for empty targetgroup or only set it if it isn't already set.